src/wazuh_db/wdb.c

```diff
@@ -643,14 +728,28 @@ void wdb_commit_old() {
         w_mutex_lock(&node->mutex);
 
         if (node->transaction && time(NULL) - node->last > config.commit_time) {
-            mdebug2("Committing database for agent %s", node->agent_id);
+            mdebug2("Committing database for agent %s", node->id);
             wdb_commit2(node);
         }
 
         w_mutex_unlock(&node->mutex);
     }
 
     w_mutex_unlock(&pool_mutex);
+
+    /* Global */
+    w_mutex_lock(&global_mutex);
+    
+    if (wdb_global != NULL) { 
+        w_mutex_lock(&db_global->mutex);
+        if (db_global->transaction && time(NULL) - db_global->last > config.commit_time) {
+            mdebug2("Committing database for %s", db_global->id);
+            wdb_commit2(db_global);
```

Comment:
Why is added this commitment for the `global.db` here?

Reply:
It is not necessary. I have removed it.